### ISSUE TITLE
Suggestion: have MathCAT add a key binding to move to the next/previous math item

### ISSUE THREAD

**Issue by NSoiffer** [NSoiffer/MathCATForPython#45]
NVDA doesn't have a way to navigate math. The current bindings using "m" are:
* m/M -- move to next/previous frame
* NVDA+shift+m -- Move mouse to current navigator object
* NVDA+alt+m --  Interact with math content
* NVDA+ctrl+m -- Brings up mouse settings dialog (not documented)
* NVDA+m -- Enable mouse tracking

"control+m" seems to be open. So maybe control+m/control+shift+m could be used?

Another option is to make it an option in the MathCAT settings dialog and let the user choose the key binding. It's more work to do that though (both an entry in the dialog, defining a method of specifying the key binding, and interpreting that binding). NVDA does have some support for adding new "input gestures" for existing commands, but "move to math" is not an option. Perhaps this code could be used in a dialog...

Anyone have suggestions?

**Comment by NSoiffer** [comment]
@ways2read suggested `=` and `shift+=`

**Comment by bhavyashah** [comment]
As a side note, I don't remember using "m" to jump by frames in a long time. Anyhow, I recognize that overwriting it may be unwise and that that is a conversation to be had within the nVDA project. All of that to say I support = and Shift+= to jump by Math block.

**Comment by NSoiffer** [comment]
Done as part of NVDA 2024.2. See [#16350](https://github.com/nvaccess/nvda/issues/16350).
